styles/variables-base.less

```diff
@@ -415,18 +415,18 @@
 
 // Item
 // ---------------------------------------
-@agate-item-border-width:             inherit;
+@agate-item-border-width:             0;
 @agate-item-content-width:            60%;
-@agate-item-centered-inline-max-width: @agate-item-inline-max-width;
+@agate-item-centered-inline-max-width: @agate-item-inline-min-width;
```

Comment:
Why we should change this?

Reply:
Yes, also check "centered" knob. It is only valid for centered. If I undo this change, when hovered, the item becomes wider. 
For inline item with centered=false the max-width is indeed @agate-item-inline-max-width.